src/routes/safe/components/Transactions/TxsTable/TxType/index.jsx

```diff
@@ -31,26 +31,30 @@ const typeToLabel = {
 }
 
 const TxType = ({ origin, txType }: { txType: TransactionType, origin: string | null }) => {
-  const isThirdPartyApp = txType === 'third-party-app'
   const [loading, setLoading] = useState(true)
   const [appInfo, setAppInfo] = useState()
+  const [forceCustom, setForceCustom] = useState(false)
 
   useEffect(() => {
     const getAppInfo = async () => {
       const parsedOrigin = getAppInfoFromOrigin(origin)
```

Comment:
could this logic be moved to `fetchTransactions` action?

Reply:
not sure to understand

Reply:
do you mean the `getAppInfo()`?